aiopyrq/unique_queues.py

```diff
@@ -134,6 +138,26 @@ async def ack_items(self, items: list) -> None:
         await pipeline.execute()
         await self._wait_for_synced_slaves()
 
+    async def can_rollback_item(self, item) -> bool:
+        """
+        Check if `item` has not been rolledback too many times
```

Comment:
je to zase jen takova drobnost, zda to trosku vic nevysvetlit s tim timoutem - proc ten max timeout.